grails-app/migrations/core/2020-04-15-user-refactor-username-add-email.groovy

```diff
@@ -0,0 +1,23 @@
+package core
+
+import com.unifina.utils.EmailValidator
+
+databaseChangeLog = {
+	changeSet(author: "kkn", id: "user-refactor-username-add-email-1") {
+		addColumn(tableName: "sec_user") {
+			column(name: "email", type: "varchar(255)")
+		}
+	}
+	changeSet(author: "kkn", id: "user-refactor-username-add-email-2") {
+		grailsChange {
+			change {
+				sql.eachRow("select username from sec_user") { row ->
```

Comment:
Can be done with one query:

`update sec_user set email = username where username like '%@%'`